src/main/java/com/google/cloud/tools/gradle/appengine/core/AppEngineCorePluginConfiguration.java

```diff
@@ -58,6 +58,19 @@ public void configureCoreProperties(
       Project project,
       AppEngineCoreExtensionProperties appEngineCoreExtensionProperties,
       String taskGroup) {
+    project
+        .getLogger()
+        .warn(
+            "WARNING: You are a using a release candidate for "
```

Comment:
I think we might need to work this first sentence another way? Does it render as:
```
You are using a release candidate for 2.0.0-rc1. Behavior....
```

Reply:
Yeah, I guess that makes it sound like a release candidate for a release candidate.